packages/example-client/src/json-server.ts

```diff
@@ -207,8 +207,9 @@ export class JsonServer {
             const response = await xhr({ url, followRedirects: 5 });
             return response.responseText;
         }
-        catch (error) {
-            return Promise.reject(error.responseText || getErrorStatusDescription(error.status) || error.toString());
+        catch (error: unknown) {
+            const err = error as Record<string, unknown>;
+            return Promise.reject(err.responseText || getErrorStatusDescription(err.status as number) || err.toString());
```

Comment:
nitpicking: May I suggest using the new `??` operator instead of `||`

Reply:
Yes, but I didn't change it in this PR. I only added:
```javascript
const err = error as Record<string, unknown>;
```

Reply:
Ok let's do it afterwards then.

We should probably have a stricter eslint config that doesn't allow using `||` on anything else than boolean

Reply:
Yes, I eslint config is actually missing. I will put this on this list.